static/styles/compose.css

```diff
@@ -135,8 +135,16 @@
 
     #private-message .to_text {
         vertical-align: middle;
-
         font-weight: 600;
+        cursor: pointer;
+
+        &:hover {
+            opacity: 0.7;
+        }
```

Comment:
This hover behavior doesn't match the other buttons in the compose box -- they all get darker on hover, not lighter.

Also we'll want to test the dark theme after fixing this.

Reply:
Fixed in most recent commit

turns out I was using dark mode and did not realize that in the light theme hover elements get darker and in dark mode they get lighter, this is now consistent with the other buttons in the compose box.